product_docs/docs/pgd/5/admin-manual/installing/03-configuring-repositories.mdx

```diff
@@ -61,40 +61,40 @@ export EDB_SUBSCRIPTION_TOKEN=<your-repo-token>
 You can add this to your `.bashrc` script or similar shell profile to ensure it's always set. 
 
 !!! Note 
-Your preferred platform may support storing this variable as a secret which can appear as an environment variable. If this is the case, don't add the setting to `.bashrc` and instead add it to your platform's secret manager.
+Your preferred platform may support storing this variable as a secret, which can appear as an environment variable. If this is the case, don't add the setting to `.bashrc`. Instea, add it to your platform's secret manager.
```

Comment:
```suggestion
Your preferred platform may support storing this variable as a secret, which can appear as an environment variable. If this is the case, don't add the setting to `.bashrc`. Instead, add it to your platform's secret manager.
```

Reply:
I dislike the disconnection of the negative action from the positive action, so instead of all these tweaks:

```suggestion
Your preferred platform may support storing this variable as a secret, which can appear as an environment variable. If this is the case, add it to your platform's secret manager, and don't add the setting to `.bashrc`. 
```